test/args.carp

```diff
@@ -1,3 +1,11 @@
+(load "Test.carp")
+(use Test)
+
 (defn main []
-  (for [i 0 (System.get-args-len)]
-    (IO.println (System.get-arg i))))
+  (with-test test
+    (assert-true test
+                 (= 1 (System.get-args-len))
+                 "System.get-args-len works as expected")
+    (assert-true test
```

Comment:
This test fails for me... Also, it's better to use `assert-equal` in that case to get a better error message.